assets/_scss/_block_styles.scss

```
@@ -66,12 +66,14 @@ body :where(.is-style-main-layout.is-layout-flex) {
 		top: 0;
 
 		@media (min-width:601px) {
-			top: var(--wp-admin--admin-bar--height);
+			top: var(--wp-admin--admin-bar--height, 0px);
 		}
 	}
 }
-// WordPress標準の position 属性。stickyは出力されるのにfixedはフロントでは出力されないため。
-*[class*="is-position-fixed"]{
+// WordPress標準の position 属性。sticky は出力されるのに fixed はフロントでは出力されないため。
```

Stylelint (17.12.0)

[error] Expected empty line before comment (scss/double-slash-comment-empty-line-before)

(scss/double-slash-comment-empty-line-before)